bin/publish.ml

```diff
@@ -85,7 +81,7 @@ let publish ?build_dir ?opam ?delegate ?change_log ?distrib_uri ?distrib_file
     match artefact with
     | `Doc -> publish_doc ~specific:specific_doc ~dry_run ~yes pkg_names pkg
     | `Distrib -> publish_distrib ?token ?distrib_uri ~dry_run ~yes pkg
-    | `Alt kind -> publish_alt ?distrib_uri ~dry_run pkg kind
+    | `Alt kind -> Deprecate_delegates.publish_alt ~dry_run pkg kind
```

Comment:
I think it makes more sense to just add a log, something like:
```suggestion
    | `Alt kind ->
      Deprecate.Delegate.artifact ();
      publish_alt ?distrib_uri ~dry_run pkg kind
```
Where `Deprecate.Delegate.artifact` would just log the right warning.

Reply:
Ok, I've done that now.